crates/metrics/src/txpool_metrics.rs

```diff
@@ -1,24 +1,97 @@
-use crate::global_registry;
-use prometheus_client::metrics::histogram::Histogram;
+use crate::{
+    buckets::{
+        buckets,
+        Buckets,
+    },
+    global_registry,
+};
+use prometheus_client::metrics::{
+    gauge::Gauge,
+    histogram::Histogram,
+};
 use std::sync::OnceLock;
 
 pub struct TxPoolMetrics {
-    pub tx_size_histogram: Histogram,
+    /// Size of transactions in the txpool in bytes
+    pub tx_size: Histogram,
+    /// Number of transactions in the txpool
+    pub number_of_transactions: Gauge,
+    /// Number of transactions pending verification, before being inserted in the txpool
+    pub number_of_transactions_pending_verification: Gauge,
+    /// Number of transactions that can be included in the next block
+    pub number_of_executable_transactions: Gauge,
+    /// Time of transactions in the txpool in seconds
+    pub transaction_time_in_txpool_secs: Histogram,
+    /// Time actively spent by transaction insertion in the thread pool
+    pub transaction_insertion_time_in_thread_pool_milliseconds: Histogram,
+    /// How long it took for the selection algorithm to select transactions
+    pub select_transaction_time_nanoseconds: Histogram,
 }
 
 impl Default for TxPoolMetrics {
     fn default() -> Self {
-        let tx_sizes = Vec::new();
+        let tx_size = Histogram::new(buckets(Buckets::TransactionSize));
+        let transaction_time_in_txpool_secs = Histogram::new(buckets(Buckets::Timing));
+        let select_transaction_time_nanoseconds =
+            Histogram::new(buckets(Buckets::TimingCoarseGrained));
```

Comment:
<img width="549" alt="image" src="https://github.com/user-attachments/assets/ba12a62f-a2f4-44a1-9935-69b92ce68af8">


I think these values are too small for selection of transaction and insertion into TxPool. The maximum values. I think we want to have: 

50 microseconds
250 microseconds
1 milliseconds
10 milliseconds
100 milliseconds
300 milliseconds (because it is 1/3 of the block execution time)
1 second
5 second

Reply:
Also you use the same bucket table for nanoseconds and milliseconds

Reply:
Buckets have been updated in https://github.com/FuelLabs/fuel-core/pull/2385/commits/58bf187b0f6d57a817b3d5a0573b02f9787be4c8 and https://github.com/FuelLabs/fuel-core/pull/2385/commits/d46468c5f2fbc0344648b3bbac6946edb36cb133